packages/widget-v2/package.json

```diff
@@ -60,6 +60,7 @@
     "@ebay/nice-modal-react": "^1.2.13",
     "@eslint/compat": "^1.1.1",
     "@initia/initia-registry": "^0.1.16",
+    "@leapwallet/cosmos-social-login-capsule-provider": "^0.0.44",
```

Comment:
this fixes yarn install, it's the only extra dep that you'd need to install manually when using yarn